v2-apache/conf/extra/httpd-ssl.conf

```diff
@@ -0,0 +1,14 @@
+Listen 443

```

Comment:
Please introduce an env var to make this port configurable as well (i.e. `SSL_PORT`)